src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,16 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL8Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/movies</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">rootroot</property>
+        <property name="show_sql">true</property>
+        <property name="format_sql">true</property>
+        <property name="hibernate.hbm2dd1.auto">update</property>
```

Comment:
```suggestion
        <property name="hibernate.hbm2dd1.auto">create-drop</property>
```

Reply:
I was looking for a way to solve the problem described above, this was one of the types of its solution.

Reply:
Use create-drop, it's not related to your exception